### PR TITLE
Documentation for Fonestorm 2.1.0

### PR DESCRIPTION
**Pull request by ubikorp**
Now includes favicon!

### CODE REVIEW

File: source/includes/_fonenumbers.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ $ curl --request GET
 ```json
 {
   "statusCode": 200,
-  "dids": [
+  "Fonenumbers": [
```
Comment (calmdev) [inline source/includes/_fonenumbers.md:21]: Lowercase `fonenumbers`

File: source/includes/_fonenumbers.md [hunk 2]
```diff
@@ -150,7 +154,7 @@ $ curl --request GET
 ```json
 {
   "statusCode": 200,
-  "did": {
+  "Fonenumber": {
```
Comment (calmdev) [inline source/includes/_fonenumbers.md:157]: Lowercase `fonenumber`

File: source/includes/_fonenumbers.md [hunk 3]
```diff
@@ -87,7 +87,7 @@ $ curl --request POST
...
```
Comment (calmdev) [inline source/includes/_fonenumbers.md:90]: Lowercase `fonenumber`

**Review by calmdev — COMMENTED** [review]
There a few responses properties that need the casing to be updated.

**Comment by calmdev** [comment]
Can I push response updates onto this branch?

**Comment by calmdev** [comment]
Actually nevermind. I will wait for this to be merged and create a new branch.

File: source/includes/_fonenumbers.md (hunk 1 above)
Comment (ubikorp) [reply source/includes/_fonenumbers.md:21]: I didn't really go through the code samples, figured I'd be checking through them when I added the other languages.

Comment (calmdev) [reply source/includes/_fonenumbers.md:21]: can we merge this? I will fix it in a different branch. All these fonenumber responses have changed.

**Pull request merged**